android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -704,23 +387,17 @@ private void populateBalance(String balance, boolean from, int responseDecimals)
             fromToBalance = Utils.getBalanceForCoinType(tokenCoin, decimals, balance);
         } catch (NumberFormatException | NullPointerException e) {
             Log.e(TAG, "Error while parsing token balance.", e);
-            clearBalance(from);
+            clearFromAssetState();
             return;
         }
 
         String text = getText(R.string.crypto_wallet_balance) + " "
                 + String.format(Locale.getDefault(), "%.4f", fromToBalance);
         textView.setText(text);
-        if (from) {
-            mConvertedFromBalance = fromToBalance;
-            if (mActivityType == ActivityType.SEND) mBtnBuySendSwap.setEnabled(fromToBalance != 0);
-        } else
-            mConvertedToBalance = fromToBalance;
-
-        if (mActivityType == ActivityType.SWAP) {
-            getSendSwapQuota(true, false);
-        }
+        mConvertedFromBalance = fromToBalance;
+        if (mActivityType == ActivityType.SEND) mBtnBuySendSwap.setEnabled(fromToBalance != 0);
```

Comment:
we can rename `mBtnBuySendSwap` to `mBtnBuySend`

Reply:
Fixed.